### PR TITLE
 Document suggested platform names #17

### LINKED ISSUE #17: Document suggested platform names
The documentation for `storage:platform` links to a chapter about the platforms, but the values given there don't correspond to the values given in examples. What platform names should be used for common/popular platforms? This would improve interoperability and clients could link to more useful information if a common set of values would have been agreed on.

### PR DESCRIPTION
**Pull request by m-mohr**
Fixes #17

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -34,16 +34,18 @@ to all assets (e.g., storage:platform=AWS if all assets are on AWS), then it sho
...
-- Azure
-- Google Cloud Platform
-- IBM
-- Oracle
+- Alibaba: `ALIBABA`
+- Amazon AWS: `AWS`
+- Microsoft Azure: `AZURE`
+- Google Cloud Platform: `GCP`
+- IBM: `IBM`
+- Oracle: `ORACLE`
```
Comment (m-mohr) [inline README.md:42]:
```suggestion
- Oracle Cloud: `ORACLE`
```

File: README.md [hunk 2]
```diff
@@ -34,16 +34,18 @@ to all assets (e.g., storage:platform=AWS if all assets are on AWS), then it sho
...
+
+The upper-cased values are meant to be used for `storage:platform`.
 
 #### Cloud Provider Storage Tiers
 
-| Minimum Duration | [Google Cloud](https://cloud.google.com/storage/docs/storage-classes) | [AWS](https://aws.amazon.com/s3/storage-classes/) | [Azure](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-storage-tiers)      | [IBM](https://cloud.ibm.com/objectstorage/create#pricing)  | [Oracle](https://www.oracle.com/cloud/storage/pricing.html) | [Alibaba](https://www.alibabacloud.com/product/oss/pricing) |
+| Minimum Duration | [Google Cloud Platform](https://cloud.google.com/storage/docs/storage-classes) | [Amazon AWS](https://aws.amazon.com/s3/storage-classes/) | [Microsoft Azure](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-storage-tiers)      | [IBM](https://cloud.ibm.com/objectstorage/create#pricing)  | [Oracle](https://www.oracle.com/cloud/storage/pricing.html) | [Alibaba](https://www.alibabacloud.com/product/oss/pricing) |
```
Comment (m-mohr) [inline README.md:48]:
```suggestion
| Minimum Duration | [Google Cloud Platform](https://cloud.google.com/storage/docs/storage-classes) | [Amazon AWS](https://aws.amazon.com/s3/storage-classes/) | [Microsoft Azure](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-storage-tiers) | [IBM Cloud](https://cloud.ibm.com/objectstorage/create#pricing)  | [Oracle Cloud](https://www.oracle.com/cloud/storage/pricing.html) | [Alibaba Cloud](https://www.alibabacloud.com/product/oss/pricing) |
```

File: README.md [hunk 3]
```diff
@@ -34,16 +34,18 @@ to all assets (e.g., storage:platform=AWS if all assets are on AWS), then it sho
...
-- AWS
-- Azure
-- Google Cloud Platform
-- IBM
-- Oracle
+- Alibaba: `ALIBABA`
+- Amazon AWS: `AWS`
+- Microsoft Azure: `AZURE`
+- Google Cloud Platform: `GCP`
+- IBM: `IBM`
```
Comment (m-mohr) [inline README.md:41]:
```suggestion
- IBM Cloud: `IBM`
```

File: README.md [hunk 4]
```diff
@@ -34,16 +34,18 @@ to all assets (e.g., storage:platform=AWS if all assets are on AWS), then it sho
 #### Providers
 Currently this document is arranged to support object storage users of the following PaaS solutions:
 
-- Alibaba
-- AWS
-- Azure
-- Google Cloud Platform
-- IBM
-- Oracle
+- Alibaba: `ALIBABA`
```
Comment (m-mohr) [inline README.md:37]:
```suggestion
- Alibaba Cloud (Aliyun): `ALIBABA`
```

**Comment by m-mohr** [comment]
Any chance to have a look? @matthewhanson

File: README.md [hunk 5]
```diff
@@ -34,16 +34,18 @@ to all assets (e.g., storage:platform=AWS if all assets are on AWS), then it sho
...
-- Azure
-- Google Cloud Platform
-- IBM
-- Oracle
+- Alibaba Cloud (Aliyun): `ALIBABA`
+- Amazon AWS: `AWS`
+- Microsoft Azure: `AZURE`
+- Google Cloud Platform: `GCP`
+- IBM Cloud: `IBM`
+- Oracle Cloud: `ORACLE`
```
Comment (duckontheweb) [inline README.md:42]: @m-mohr Should we also document the `OTHER` value here?

Comment (m-mohr) [reply README.md:42]: Yes, that makes sense.

Comment (m-mohr) [inline README.md:42]:
```suggestion
- Oracle Cloud: `ORACLE`
- All other PaaS solutions: `OTHER`
```

**Review by duckontheweb — APPROVED** [review]

**Pull request merged**